watertap/unit_models/tests/test_gac.py

```diff
@@ -478,3 +562,80 @@ def test_costing_max_gac_ref_robust(self, gac_frame_robust):
         assert value(m.fs.unit.costing.bed_mass_gac_ref) == (
             pytest.approx(value(m.fs.costing.bed_mass_gac_max_ref), 1e-5)
         )
+
+    @pytest.fixture(scope="class")
+    def gac_frame_robust_upscale(self):
+        m = ConcreteModel()
+        m.fs = FlowsheetBlock(default={"dynamic": False})
+
+        # specs copied over from gac_frame_robust to isolate testing of 100x upscaled flow
+        m.fs.properties = DSPMDEParameterBlock(
+            default={"solute_list": ["TCE"], "mw_data": {"H2O": 18e-3, "TCE": 131.4e-3}}
+        )
+
+        m.fs.unit = GAC(
+            default={
+                "property_package": m.fs.properties,
+                "film_transfer_coefficient_type": "calculated",
+                "surface_diffusion_coefficient_type": "calculated",
+            }
+        )
+
+        # feed specifications
+        m.fs.unit.process_flow.properties_in[0].pressure.fix(
+            101325
+        )  # feed pressure [Pa]
+        m.fs.unit.process_flow.properties_in[0].temperature.fix(
+            273.15 + 25
+        )  # feed temperature [K]
+        m.fs.unit.process_flow.properties_in[0].flow_mol_phase_comp["Liq", "H2O"].fix(
+            1000 * 824.0736620370348
+        )
+        m.fs.unit.process_flow.properties_in[0].flow_mol_phase_comp["Liq", "TCE"].fix(
+            1000 * 5.644342973110135e-05
+        )
+        m.fs.unit.process_flow.properties_in[0].flow_vol_phase["Liq"]
+        m.fs.unit.process_flow.properties_in[0].conc_mass_phase_comp
+
+        m.fs.unit.conc_ratio_replace.fix(0.80)
+        m.fs.unit.freund_k.fix(1062e-6 * (1e6**0.48))
+        m.fs.unit.freund_ninv.fix(0.48)
+        m.fs.unit.ebct.fix(10 * 60)
+        m.fs.unit.bed_voidage.fix(0.44)
+        m.fs.unit.particle_porosity.fix(0.641)
+        m.fs.unit.particle_dens_app.fix(803.4)
+        m.fs.unit.particle_dia.fix(0.001026)
+        m.fs.unit.velocity_sup.fix(5 / 3600)
+        m.fs.unit.molal_volume.fix(9.81e-5)
+        m.fs.unit.tort.fix(1)
+        m.fs.unit.spdfr.fix(1)
+        m.fs.unit.sphericity.fix(1.5)
+        m.fs.unit.a0.fix(0.8)
+        m.fs.unit.a1.fix(0)
+        m.fs.unit.b0.fix(0.023)
+        m.fs.unit.b1.fix(0.793673)
+        m.fs.unit.b2.fix(0.039324)
+        m.fs.unit.b3.fix(0.009326)
+        m.fs.unit.b4.fix(0.08275)
```

Comment:
Regarding the Stanton and throughput equation parameters, how are these determined? That is, what are they primarily dependent on?

Reply:
The reason I ask is because I wonder if these should be kept as Vars in the unit model and then fixed, like we do with certain parameters in property models. Then these default values would used as the initial vals. The user could then overwrite them as needed by fixing new values to them (or potentially using parameter estimation down the line). Not sure if we'd want to take that route or not as I anticipate that these parameters are highly specific, but it could help reduce the number of vars that need to be fixed by the user that might not have this information at the ready.

Reply:
Both a#s and b#s are functions of 1/n (fixed and highly specific to the contaminants) and Bi (calculated) already present in the model. These parameters are tabulated alongside the constant pattern model development (Hand et al., 1984) which is relatively available. In literature, parameter values are referenced and linearly interpolated as needed to get their value. Concerning their nonlinearity with 1/n and Bi, I am not sure of the goodness of fit to any simple regression formula which would remove the need to fix and introduce new equations.